docs-web/configuration/global-configuration/configmap-resource.md

```diff
@@ -205,7 +205,22 @@ See the doc about [VirtualServer and VirtualServerRoute resources](/nginx-ingres
    * - ``log-format``
      - Sets the custom `log format <https://nginx.org/en/docs/http/ngx_http_log_module.html#log_format>`_ for HTTP and HTTPS traffic. For convenience, it is possible to define the log format across multiple lines (each line separated by ``\n``). In that case, the Ingress Controller will replace every ``\n`` character with a space character. All ``'`` characters must be escaped.
      - See the `template file <https://github.com/nginxinc/kubernetes-ingress/blob/master/internal/configs/version1/nginx.tmpl>`_ for the access log.
-     - 
+     - ```kind: ConfigMap
```

Comment:
This is invalid RST syntax and doesn't render correctly. Also, note that all other examples link to some example in https://github.com/nginxinc/kubernetes-ingress/tree/master/examples/ which will give you access to markdown syntax instead.

I think it will be better to add an example there instead. - What do you think?

I assume that's what @pleshakov had in mind from https://github.com/nginxinc/kubernetes-ingress/pull/1231/files#r525617497